### DISCUSSION TITLE
dbus-serialbattery coexistence with bluetooth sensors, e.g. Ruuvi?

### DISCUSSION THREAD

**Discussion by aforjeh79** [Louisvdw/dbus-serialbattery#997] [Q&A]
Hi All

I'm new to VenusOS, and installed dbus-serialbattery (latest, v1.0.20230531) and a ruuvi tag on a new installation (on an RPI 3 B+). I tested various Venus versions, primarily 3.20 and now 3.30.

I had frequent disconnections of both the Ruuvi tag and my BMS (JDB/Fogstar Drift via Bluetooth) on all version combinations, typically at least once a day, or sometimes only after a few hours or less.

Before I spend time investigating logs etc, I've simply removed dbus-serialbattery and left the Ruuvi tag to rule out any signal/raspberry power/stability type issues. It has been rock solid for 10 days without dbus-serialbattery.

My battery is a commercially produced one, with a bluetooth connection only to the BMS - opening it up to access the UART port would be messy/difficult and invalidate my warranty.

So my question is - is it expected that dbus-serialbattery would be cause instability in the native Venus support for BlueTooth sensors such as Ruuvi/Mopeka?

And if so - is it considered a bug/work in progress or simply expected behaviour?

Thanks

Aaron

**Comment by NorthTown2022** [comment]
Were you using Ruuvi sensors before installing dbus-serialbattery? There is some good info and a list of recommended bluetooth dongles at Victron Community [here](https://community.victronenergy.com/questions/140360/cerbo-gx-usb-bluetooth-how-can-i-make-sure-that-ce.html) if you haven't already come across it that might help. I have a few Ruuvis and have not noticed any difference however I'm using a serial connection with dbus-serialbattery.

**Comment by aforjeh79** [comment]
All of this setup is new, so no I was not using Ruuvi before. As I said, with dbus-serialbattery removed Ruuvi is perfectly stable. So I don’t have reason to suspect there is any issue with the bluetooth adapter itself..

If you are using serial I wouldn’t expect that would conflict with BT… in my case I must use BT, and I’m wondering if dbus-serialbattery is known to conflict with stuff like Ruuvi…
